strr-web/components/bcros/form-section/property/Details.vue

```diff
@@ -83,14 +78,64 @@
             :placeholder="t('createAccount.propertyForm.ownershipType')"
             :options="ownershipTypes"
             option-attribute="name"
-            class="w-full"
-            style="color: #1a202c; /* text-gray-900 */ dark:text-white; /* Override with dark mode text color */"
             :error="ownershipTypeError"
+            style="color: #1a202c; /* text-gray-900 */ dark:text-white; /* Override with dark mode text color */"
             @blur="emit('validateOwnership')"
             @change="emit('validateOwnership')"
           />
         </UFormGroup>
       </div>
+
+      <div class="flex flex-row justify-between w-full mb-[40px] mobile:mb-[16px]">
+        <UFormGroup name="typeOfSpace" class="d:pr-[16px] flex-grow" :error="typeOfSpaceError">
+          <USelect
+            v-model="typeOfSpace"
+            aria-label="type of space"
+            :placeholder="t('createAccount.propertyForm.typeOfSpace')"
+            :options="typeOfSpaceOptions"
+            option-attribute="name"
+            style="color: #1a202c; /* text-gray-900 */ dark:text-white; /* Override with dark mode text color */"
+            @blur="emit('validateTypeOfSpace')"
+            @change="emit('validateTypeOfSpace')"
+          />
+        </UFormGroup>
+      </div>
+      <div class="flex flex-row justify-between w-full mb-[40px] mobile:mb-[16px]">
+        <UFormGroup name="isOnSameProperty" class="d:pr-[16px] flex-grow">
+          <USelect
+            v-model="isOnSameProperty"
+            aria-label="is on same property"
```

Comment:
If `aria-label` is truly required here, it should come from translations file en.json via useTranslation() composable. And same for other labels.